### PR TITLE
[T4A6][T15-B3]Bartłomiej Woźniak

### PR DESCRIPTION
**Pull request by ghost**
Add a UtilsTest.java containing JUnit tests for the Utils class.

### CODE REVIEW

**Comment by ghost** [comment]
Ready for review

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import org.junit.Test;
+
+import seedu.addressbook.commands.IncorrectCommand;
+import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.parser.Parser;
+
+public class UtilsTest {
+
+    @Test
+    public void elementsAreUnique_elementsAreUnique() {
```
Comment (nishantbudhdev) [inline test/java/seedu/addressbook/common/UtilsTest.java:18]: Mention the expected return value in the method name.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    	ArrayList<Integer> elements = new ArrayList<Integer>();
```
Comment (nishantbudhdev) [inline test/java/seedu/addressbook/common/UtilsTest.java:19]: You can move common initialisations/ code to a setup/ new method.

**Comment by nishantbudhdev** [comment]
@bartekwozniak93 Good work.

Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**